posthog/api/test/dashboards/test_dashboard.py

```diff
@@ -428,10 +428,8 @@ def test_dashboard_insight_tiles_can_be_loaded_correct_context(self):
         response = self.dashboard_api.get_dashboard(dashboard_id)
         self.assertEqual(len(response["tiles"]), 1)
         self.assertEqual(len(response["tiles"]), 1)
-        item_insight = response["tiles"][0]
         tile = response["tiles"][0]
 
-        assert item_insight["filters_hash"] == tile["filters_hash"]
         assert tile["insight"]["id"] == insight_id
```

Comment:
The interesting thing about removing this is it makes it clear how lazy I was with this test 😅 

What we care about here is what filters have been applied to the insights... We were checking the filters hash as a proxy for that. So would need to assert on something like `response["tiles"][0]["insight"]["filters"]["date_from"] == "-14d"`

Reply:
Thank you, will update!